### ISSUE TITLE
Vibrancy while keeping macOS window border

### ISSUE THREAD

**Issue by probablykasper** [amrbashir/tauri-plugin-vibrancy#49]
Pretty sure you have to make windows transparent to enable vibrancy currently. On macOS dark mode, windows have a bright border in order to better see windows. When you make a window transparent, that border is removed. Ideally there would be vibrancy while keeping the window border.

**Comment by fr3fou** [comment]
Any updates or possible workarounds?

**Comment by ayangweb** [comment]
Hey 👋, have you solved this problem? If you solved it, can you share the solution idea?

**Comment by probablykasper** [comment]
No, but feel free to open a PR. You can investigate the code and look for macOS documentation and similar projects

**Comment by ayangweb** [comment]
> No, but feel free to open a PR. You can investigate the code and look for macOS documentation and similar projects

OK, I'll look for another solution

**Comment by insd47** [comment]
found some workaround!

the bright border disappears when window background is completely transparent,
so I applied tiny alpha value for ns_window.

you can use tauri 2's window effects config for vibrancy. here is an example code.
```rust
    // Window config for macOS
    #[cfg(target_os = "macos")]
    let win_builder = win_builder
        .title_bar_style(TitleBarStyle::Overlay)
        .hidden_title(true)
        .background_color(Color(0, 0, 0, 1)) // <-
        .effects(EffectsBuilder::new()
            .effects(vec![Effect::Sidebar])
            .build())
        .traffic_light_position(LogicalPosition::new(15.0, 20.0));
```

<img width="1602" alt="Image" src="https://github.com/user-attachments/assets/5ab0275d-680e-4aec-9ae1-8d569786d2a8" />

(sorry for my bad english)

**Comment by rohanrehman** [comment]
@insd47 can you share to a public repo, current docs are missing some steps in setting up window-vibrancy.

**Comment by rohanrehman** [comment]
@insd47 Thanks for this! I was using window-vibrancy 0.60.

But basically you aren't using the vibrancy code correct?
